### PR TITLE
Wishlist Screen

### PR DESCRIPTION
**Pull request by m26unkwn**
Added The wishlist Screen

### CODE REVIEW

File: styles/product.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.main-wrapper {
+.main-wrapper-product {
   display: grid;
```
Comment (PuneetP16) [inline styles/product.css:2]: It's a nice modular approach to class naming. It will make it easier to apply changes in the long run as well.

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge! 
Just a suggestion though, you could reduce the size of close button a bit everything else looks good.

image

File: styles/product.css (hunk 1 above)
Comment (m26unkwn) [reply styles/product.css:2]: Thanks @PuneetP16  I will reduce the size of the close button.

File: screens/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+                      />
+                    </g>
+                  </svg>
+                </button>
+              </div>
+              <div class="pd-content">
+                <h1 class="pd-heading card-heading">Keychron K2 V-2</h1>
+                <p class="pd-desc">mechanical keyboard</p>
+                <div class="pd-price">
+                  <p class="crnt-price">$200</p>
```
Comment (ashishkatiyar2404) [inline screens/wishlist.html:199]:
```suggestion
                  <p class="current-price">$200</p>
```

File: screens/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,618 @@
...
```
Comment (ashishkatiyar2404) [inline screens/wishlist.html:329]:
```suggestion
                  <p class="current-price">$200</p>
```

File: screens/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,618 @@
...
```
Comment (ashishkatiyar2404) [inline screens/wishlist.html:394]:
```suggestion
                  <p class="current-price">$200</p>
```

**Review by ashishkatiyar2404 — APPROVED** [review]
Your UI looks good. just a suggestion use full images of dev t-shirt

**Pull request merged**